cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -487,14 +488,14 @@
 #   the unittests
 
 # Valid anndata
-adata = anndata.AnnData(X=sparse.csr_matrix(X), obs=good_obs, uns=good_uns, obsm=good_obsm, var=good_var)
+adata = anndata.AnnData(X=X, obs=good_obs, uns=good_uns, obsm=good_obsm, var=good_var)
```

Comment:
These should be copies. Don't trust my syntax bellow. 
```suggestion
adata = anndata.AnnData(X=X.copy(), obs=good_obs, uns=good_uns, obsm=good_obsm, var=good_var)
```